### ISSUE TITLE
Add more site info into the model

### ISSUE THREAD

**Issue by ibaldin** [fabric-testbed/InformationModel#164]
- Site IPv4/IPv6 status
- Site hosting org (not the same as location)
- PTP status
- ...

**Comment by yaxue1123** [comment]
Example:
```
{
  "name": "RENCI",
  "acronym": "RENC",
  "hosting": "University of North Carolina-Chapel Hill RENCI", 
  "ipv": "4/6",
  "location": "Chapel Hill, NC",
  ...
}
```

**Comment by ibaldin** [comment]
PTP status will be available as of FIM/CF 1.5.0
